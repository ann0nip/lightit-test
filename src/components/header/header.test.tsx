import { MemoryRouter as Router } from 'react-router-dom';
import { render } from '@testing-library/react';
import Header from './header.component';

describe('Header Component', () => {
  it('should take a snapshot', () => {
    const { asFragment } = render(
      <Router>
        <Header />
      </Router>,
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
