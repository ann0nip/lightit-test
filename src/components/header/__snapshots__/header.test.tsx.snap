// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should take a snapshot 1`] = `
<DocumentFragment>
  <header
    class="flex items-center justify-between flex-wrap bg-gray-800 p-4 px-32 h-[10%]"
  >
    <a
      href="/"
    >
      <svg
        class="fill-white"
      >
        logo.svg
      </svg>
      <h1
        class="text-gray-200"
      >
        Company Name
      </h1>
    </a>
    <div
      class="border-[#979797] relative cursor-pointer"
      data-testid="dropdown-btn"
    >
      <div
        class="w-56 bg-white h-[41px] rounded-lg drop-shadow-input pl-3 focus:outline-0 focus:drop-shadow-none transition relative flex items-center justify-between"
      >
        <span>
          Guardar y salir
        </span>
        <div
          class="z-10 border-l"
        >
          <img
            alt="arrow down"
            class="mx-2 -rotate-90"
            src="Activo.png"
          />
        </div>
      </div>
    </div>
  </header>
</DocumentFragment>
`;
